### ISSUE TITLE
Issue with "nothing found message" in custom search results template

### ISSUE THREAD

**Issue by thetooftoof** [elementor/elementor#7347]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
Hello,
I've set a custom search result page with the archive template, as it's done in this video (https://www.youtube.com/watch?v=kPAUEShgdoo). I've set the archive posts widget in 4 columns, which works great when results show up, but when nothing is found, the "nothing found message" show up compressed in the first column instead of showing up in full width, or at least center in the page. I think if you can set the "nothing found message" appears overwriting the columns number to just 1, it would work.
I hope you can fix it, let me know when it's done.
Bye

![capture d ecran 2019-03-06 a 22 55 50](https://user-images.githubusercontent.com/48303330/53927436-af64bc80-4065-11e9-81e5-58715428ed13.jpg)
![capture d ecran 2019-03-06 a 22 55 16](https://user-images.githubusercontent.com/48303330/53927438-af64bc80-4065-11e9-9938-4d87a009cf08.jpg)

## Steps to reproduce
1-Make a custom search page with the archive template
2-Set the archive posts widget to 4 columns (or anything more than 1)
3-Make a search preview with a nothing found query to display the "nothing found message"
4- see that the message appear contained in the first column only

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
-MacOSX high sierra version 10.13.6
-Chrome Version 72.0.3626.119 (Build officiel) (64 bits)
-Wordpress versión 5.1–es_ES
-Elementor 2.5.3
-Elementor Pro 2.4.6

```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 84.0, Revision 47234b3 v5.6.40-84.0
	PHP Version: 7.1.26
	PHP Max Input Vars: 3000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.1
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: es
	Timezone: America/Santiago
	Debug Mode: Inactive

== Theme ==
	Name: GeneratePress Child
	Version: 2.1.4.1541815674
	Author: Tom Usborne
	Child Theme: Yes
	Parent Theme Name: GeneratePress
	Parent Theme Version: 2.2.2
	Parent Theme Author: Tom Usborne

== User ==
	Role: administrator
	WP Profile lang: es_ES
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.119 Safari/537.36

== Active Plugins ==
	Contact Form 7
		Version: 5.1.1
		Author: Takayuki Miyoshi

	Duplicator
		Version: 1.3.8
		Author: Snap Creek

	Elementor
		Version: 2.5.3
		Author: Elementor.com

	Elementor Pro
		Version: 2.4.6
		Author: Elementor.com

	GP Premium
		Version: 1.7.8
		Author: Tom Usborne

	Jet Elements For Elementor
		Version: 1.15.4
		Author: Zemez

	JetTricks
		Version: 1.1.8
		Author: Zemez

	NP Quote Request WooCommerce
		Version: 1.8.112
		Author: Neah Plugins

	Regenerate Thumbnails
		Version: 3.1.0
		Author: Alex Mills (Viper007Bond)

	SG Optimizer
		Version: 5.1.1
		Author: SiteGround

	WooCommerce
		Version: 3.5.5
		Author: Automattic

	WooCommerce Checkout Manager
		Version: 4.2.5
		Author: Visser Labs

== Log ==
	: 
Log: showing 4 of 42019-02-12 12:00:25 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.3.4',
  'to' => '2.4.6',
)]
2019-02-12 12:09:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.1.11',
  'to' => '2.4.4',
)]
2019-03-06 13:30:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.6',
  'to' => '2.5.3',
)]
2019-03-06 13:32:53 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.4',
  'to' => '2.4.6',
)]

</details>

**Comment by shilo-ey** [comment]
Hi @thetooftoof 

We cannot reproduce your issue. 
![image](https://user-images.githubusercontent.com/45009641/53939800-cc78aa00-40bc-11e9-8697-7753d2e9879e.png)

The "Nothing Found" message origins from the Posts widget.
Please make sure your Posts widget is in on column per section layout.

Thanks.

**Comment by thetooftoof** [comment]
is it in column. but when you put 4 columns in the layout of the widget, the message appears on the first column...
![capture d ecran 2019-03-07 a 05 17 28](https://user-images.githubusercontent.com/48303330/53942152-ca9cef80-4098-11e9-9b2a-fbe6a90a6af9.jpg)

**Comment by jamiemlaw** [comment]
@shilo-ey I'm afraid I've encountered the same problem recently as well. When there are search results, I want them to display in three columns, but when nothing is found, I want the 'nothing found' text to appear in one column. Is there something I'm missing?

![Screenshot 2019-08-19 at 13 52 31](https://user-images.githubusercontent.com/956081/63267534-2a273a00-c28a-11e9-8df6-43a78684defc.png)

I'm using Elemenor 2.6.8 and Elementor Pro 2.6.2

> Please make sure your Posts widget is in on column per section layout.

I'm using the Archive Posts widget to display the search results. I can see there's a Posts widget as well, but it looks like it's suited for posts with a fixed query, not search results. Which widget should I be using?

I've tried doing things like going into the preview settings, searching for something that doesn't exist, and settings the number of columns for that section to 1; but when I go back to a regular search has results, the results also appear in one column.

**Comment by shilo-ey** [comment]
Thanks for reporting. we will take a look at this issue.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.7

Feel free to update

Thanks!

**Comment by jman057** [comment]
The issue is persisting for us, with Elementor Pro version 2.8.2, where the Woocommerce "no products" alert is appearing in the first column, rather than above. It looks terrible. I quickly fixed with a simple CSS hack.

For others experiencing this issue, a quick fix seems to be to add this to your themes customer CSS:

`.no-products.woocommerce-info{
	position:absolute;
}`
